### PR TITLE
i18n: import new strings

### PR DESCRIPTION
**Pull request by brendankenny**
mostly getting back the moved `There may be stored data affecting loading performance...` warning and a translated "View Treemap" button

### CODE REVIEW

**Comment by connorjclark** [comment]
I'm also going to do an import later this evening just before release, to get any last minute strings, FWIW.

File: lighthouse-core/lib/i18n/locales/de.json [hunk 1]
```diff
@@ -1080,13 +1080,13 @@
     "message": "Der Messwert \"Erster CPU-Leerlauf\" gibt an, wann die Aktivität des Hauptthreads der Seite das erste Mal gering genug ist, um Eingaben zu verarbeiten.  [Weitere Informationen.](https://web.dev/first-cpu-idle/)"
   },
   "lighthouse-core/audits/metrics/first-meaningful-paint.js | description": {
-    "message": "\"Inhalte weitgehend gezeichnet\" gibt an, wann die Hauptinhalte einer Seite sichtbar sind. [Weitere Informationen.](https://web.dev/first-meaningful-paint/)"
+    "message": "\"First Meaningful Paint\" gibt an, wann die Hauptinhalte einer Seite sichtbar sind. [Weitere Informationen.](https://web.dev/first-meaningful-paint/)"
```
Comment (patrickhulce) [inline lighthouse-core/lib/i18n/locales/de.json:1083]: ah yes, thank goodness we're perfecting our FMP strings in time for I/O 2021 😆

**Review by patrickhulce — APPROVED** [review]

**Comment by brendankenny** [comment]
> I'm also going to do an import later this evening just before release, to get any last minute strings, FWIW.

Just trying to avoid pipeline stalls. I can leave this open and run it again this afternoon on the off chance that a locale or two has a new string.

**Comment by connorjclark** [comment]
sgtm, Thanks for the assist!

**Comment by brendankenny** [comment]
re-ran after the tc dump and there aren't any new strings (basically waiting on the `lighthouse-treemap/*` translations from #12441 and the moved page load warnings from #12469, but those landed on Friday and today, so were unlikely to come through yet)

**Pull request merged**